content/_guides/bots.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Network bots
+category: chanop
+credits: web7
```

Comment:
Is this document really based on existing content from Web-7.0, or is it original text?

Reply:
i copied a file and adapted it